docs/release_notes.md

```diff
@@ -6,6 +6,12 @@ For a list of all changes in this release, see the [full changelog](https://gith
 
 ### Features
 
+- Added three new metrics:
+	- Percent bias (PBIAS) metric: `scores.continuous.pbias`. See [PR #639](https://github.com/nci/scores/pull/639) and [PR #655](https://github.com/nci/scores/pull/655).
+	- Threshold weighted continuous ranked probability score (twCRPS) for ensembles: `scores.probability.tw_crps_for_ensemble`. See [PR #644](https://github.com/nci/scores/pull/644).
+	- Tail threshold weighted continuous ranked probability Score (twCRPS) for ensembles: `scores.probability.tail_tw_crps_for_ensemble`. See [PR #644](https://github.com/nci/scores/pull/644).
+- Adapted the FIxed Risk Multicategorical (FIRM) score (`scores.categorical.firm`) to allow it to take a mulitdimensional array (xr.DataArray) of thresholds. This allows the FIRM score to be used with categorical thresholds that vary across the domain. See [PR #661](https://github.com/nci/scores/pull/661).
```

Comment:
@nicholasloveday @rob-taggart Is the dot point about FIRM accurate and reasonably explained? If this requires any changes, please let me know.